packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -35,14 +35,17 @@ const Autocomplete = React.memo(props => {
         dropdown_offset,
         error,
         has_updating_list = true,
+        hide_list = false,
         input_id,
         is_alignment_top,
         list_items,
         list_portal_id,
         onHideDropdownList,
         onItemSelection,
         onScrollStop,
+        onShowDropdownList,
```

Comment:
needed onShowDropdownList to help with displaying the close circle icon as when you keep clicking the autocomplete, it won't show the icon after multiple clicks.